File: app/src/main/java/kr/com/misemung/vo/AddrInfo.java
package kr.com.misemung.vo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by kjh on 2019-01-21.
 */

public class AddrInfo implements Serializable {

    @SerializedName("address_name")
    @Expose
    private String address_name;
    @SerializedName("address_type")
    @Expose
    private String address_type;
    @SerializedName("road_address")
    @Expose
    private String road_address;
    @SerializedName("x")
    @Expose
    private String x;
    @SerializedName("y")
    @Expose
    private String y;

    public String getAddress_name() {
        return address_name;
    }

    public void setAddress_name(String address_name) {
        this.address_name = address_name;
    }

    public String getAddress_type() {
        return address_type;
    }

    public void setAddress_type(String address_type) {
        this.address_type = address_type;
    }

    public String getRoad_address() {
        return road_address;
    }

    public void setRoad_address(String road_address) {
        this.road_address = road_address;
    }

    public String getX() {
        return x;
    }

    public void setX(String x) {
        this.x = x;
    }

    public String getY() {
        return y;
    }

    public void setY(String y) {
        this.y = y;
    }
}

File: app/src/main/java/kr/com/misemung/vo/AddrInfo.kt
package kr.com.misemung.vo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

/**
 * Created by kjh on 2019-01-21.
 */
class AddrInfo : Serializable {
    @SerializedName("address_name")
    @Expose
    var address_name: String? = null

    @SerializedName("address_type")
    @Expose
    var address_type: String? = null

    @SerializedName("road_address")
    @Expose
    var road_address: String? = null

    @SerializedName("x")
    @Expose
    var x: String? = null

    @SerializedName("y")
    @Expose
    var y: String? = null
}